### PR TITLE
Add logo and sponsors

### PR DESCRIPTION
**Pull request by moshe742**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,6 +128,14 @@ <h3>חסות כסף</h3>
                 </a>
             </div>
           </div-->
+          <div class="bronze">
+            <h3>חסות ארד</h3>
+            <div class="sponsor">
+              <a href="https://www.commagain.org/" target="_blank">
+                <img src="sponsor_logos/cooagain_logo.png" alt="לוגו של commagain">
```
Comment (tbrisker) [inline index.html]: nitpick: the file seems to have an incorrect name

**Review by tbrisker — CHANGES_REQUESTED** [review]
The commagain logo is huge, taking up half the screen, and has some weird border in it (i think from the original file).

File: index.html (hunk 1 above)
Comment (moshe742) [reply index.html]: size fixed and the border too

Comment (tomer) [reply index.html]: @tbrisker Is this commit rejection still an issue?

Comment (tbrisker) [reply index.html]: no, the rejection was for the size, this was just a nitpick about giving the file the wrong name.

Comment (moshe742) [reply index.html]: @tomer @tbrisker can you check that the rejection is not relevant anymore and merge?

**Review by tbrisker — DISMISSED** [review]

**Comment by tbrisker** [comment]
Removed my rejection so it doesn't block merging, but i did not review. Looks like there are also merge conflicts.

**Review by tbrisker — APPROVED** [review]
This is quickly becoming a monster PR. I'm going to merge now so no more commits get added here and we can iterate on any issues afterwards. In the future please try to make your PRs confined to a single issue in a single commit so they are easier to review and merge.

**Pull request merged**